x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/process/MlMemoryTracker.java

```diff
@@ -131,11 +130,6 @@ public void stop() {
         logger.debug("ML memory tracker stopped");
     }
 
-    @Override
-    public String executorName() {
```

Comment:
We use this executor to run `MlMemoryTracker#refresh` and `MlMemoryTracker #iterateAnomalyDetectorJobTasks`, are you sure it's ok to run them on the `SAME` thread now?

Reply:
Great catch! Addressed in https://github.com/elastic/elasticsearch/pull/59932/commits/d30872145f375f109ed20e2d26db6946436c3b4e